drivers/nfs.py

```diff
@@ -65,8 +65,7 @@ def check_server_tcp(server, nfsversion=DEFAULT_NFSVERSION):
     False otherwise.
     """
     try:
-        sv = get_supported_nfs_versions(server)
-        return (True if nfsversion in sv else False)
+        return get_supported_nfs_versions(server)
```

Comment:
This appears to change the semantics of this function, which probably means that it no longer does the right thing if called with nfsversion (which it is on NFSSR.py line 116). Although I can't actually see how the return code is used there...